index.Rmd

```diff
@@ -1,105 +1,594 @@
+
+<br>
+
+Nymphal <i> A. phagocyotphilum </i> zero-inflated model:
+```{r nymphANA_sig_coeff}
+nymph_ANA_reg$coefficients$zero
+```
+
+<br>
+
+Nymphal <i> B. microti </i> count model:
+```{r nymphBm_sig_coeff}
+nymph_Bmic_reg$coefficients$count
+```
+
+<br>
+
+Nymphal <i> B. miyamotoi </i> count model:
+```{r nymphBmiya_sig_coeff}
+nymph_Bmiya_reg$coefficients$count
+```
 # Conclusions
 
-[~200 words]
+Tickborne pathogen prevalence in NYS is an ever-increasing problem. NYSDOH Vector Ecology Laboratory tick sampling indicates that pathogen prevalence for <i> B. burgdorferi </i>, <i> A. phagocytophilum </i>, <i> B. microti </i>, and <i> B. miyamotoi </i> have been increasing over time. Increases in pathogen prevalence provides a warning for increased risk of tickborne disease for NYS citizens. Although the prevalence of each pathogen, and thus, risk for disease differs by county, the general trend indicates that tickborne disease will be an issue for years to come in NYS. The interactive map in this project provides a user-friendly ability to navigate publicly available data to determine the potential risk for tickborne pathogen exposure in your county. 
+
+One extremely important factor in determining risk for tickborne pathogen exposure is tick habitat.< I. scapularis </i> prefer deciduous and mixed forest ecosystems, making tick prevention methods in these ecosystems an important topic for preventing tickborne disease in the future. This analysis aimed to determine if the percent of a county covered by these types of forest plays any role in pathogen prevalence rates. The method behind this thinking is that the more available area for tick populations to expand dually provides tickborne pathogens the ability to expand into these areas. Coincidentally, preliminary results from the correlation tests and ZINB modeling show that the percent of a county covered by mixed or deciduous forest is either not directly related to tickborne pathogen prevalence, or actually shows a decreased risk for pathogen prevalence. There are two likely explanations for these results. The first is that aggregating forest cover to large spatial areas (counties in this study), do not represent the microvariations in tick habitat that allow populations to proliferate. Smaller spatial scales may provide a more accurate representation of the relationship between land cover and pathogen prevalence. Another explanation is that there is sampling bias in these data. The NYSDOH is tasked with protecting the health of the public, and thus, sampling is generally conducted: 
+
+1. Where the greatest risk for tick-human interaction is.
+2. Where the density of tick populations is high enough to get a good sample size.
+
+The first bias would cause more data to be gathered around suburban areas due to the larger number of individuals at risk. Counties with more suburban areas will likely have a lower percentage of forest cover, thus biasing the data.
```

Comment:
good interpretation.  This is known as an ecological fallacy (aggregation can affect models like this)